lsp-server/src/main/scala/org/alephium/ralph/lsp/server/RalphLangServer.scala

```diff
@@ -47,6 +48,7 @@ object RalphLangServer extends StrictImplicitLogging {
         client = Some(client),
         listener = Some(listener),
         pcStates = PCStates.empty,
+        searchCache = SearchCache(maxWorkspaces = 1),
```

Comment:
just to be sure I understood, we start with `1` but we might test with other value when tackling [this TODO](https://github.com/alephium/ralph-lsp/blob/568e96af2436f54796d86425c01651f2a60cfb82/presentation-compiler/src/main/scala/org/alephium/ralph/lsp/pc/search/cache/SearchCache.scala#L20)?